public/client-v2/src/services/capiQuery.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { qs } from '../util/qs';
+
+const API_BASE = 'https://content.guardianapis.com/';
+
+const capiQuery = (apiKey: string) => ({
+  search: async (params: Object) => {
+    const response = await fetch(
+      `${API_BASE}search${qs({
+        ...params,
+        'api-key': apiKey
+      })}`
+    );
+
+    return response.json();
```

Comment:
I was looking at the checks the old app does on capi results and it checks that an article has in an internalPageCode before showing it in the dropdown - we should probably repeat this check.

Reply:
Is that a Front API specific thing? I don't think I've ever seen it in CAPI but yeah, we should definitely do that ... do you know what it represents?